tests/md_indent_test.py

```diff
@@ -0,0 +1,31 @@
+from rdm.render import md_indent
```

Comment:
Nice job writing these tests! Did you feel like they helped you think through the problem at all?

Reply:
They absolutely did. I never would have foreseen the case of clearing off headings and what that might entail.